app/routes/titiler/algorithms/dist_alerts.py

```diff
@@ -0,0 +1,27 @@
+from collections import OrderedDict, namedtuple
+from .alerts import Alerts
+
+from app.models.enumerators.alerts_confidence import DeforestationAlertConfidence
+
+
+class DISTAlerts(Alerts):
+    title: str = "Land Disturbunce (DIST) Alerts"
+    description: str = "Decode and visualize DIST alerts"
+
+    Colors: namedtuple = namedtuple("Colors", ["red", "green", "blue"])
+    DeforestationAlert: namedtuple = namedtuple(
+        "DeforestationAlert", ["confidence", "colors"]
```

Comment:
Since there's just a single definition of that in alerts.py now, I ended up calling it `AlertConfig` and also removed `deforestation` from a couple of variables that are common to both alert systems.